http-gateway/swagger.yaml

```diff
@@ -442,6 +442,24 @@ paths:
           $ref: '#/components/responses/badRequest'
         401:
           $ref: '#/components/responses/unauthorized'
+    delete:
```

Comment:
I don't like much that endpoints retrieves all pending command, but you can cancel only specific type of pending command. I would exclude it and create separate endpoint for pending metadata updates.